package.json

```diff
@@ -7,11 +7,11 @@
     "@material-ui/icons": "^3.0.2",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
+    "react-rating": "^1.6.2",
     "react-scripts": "2.1.5"
   },
   "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
```

Comment:
Oh wait, never mind that. Since `react-scripts build` generates a `build` folder with `index.html` and assets under `build/static/js` that have cache-busting names. So that won't work for library components.